src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/RemoteBuildConfiguration.java

```diff
@@ -856,6 +856,16 @@ public String getConsoleOutput(String urlString, String requestType, AbstractBui
                 is = connection.getErrorStream();
             }
             
+            // if the error stream is empty, which can happen, and we do not want to access it by handling it over to the BufferedReader in the next lines
+            // Nevertheless of scope we need to call the function twice as in try block defined members are not visible to outside
+            try {
+                BufferedReader rd = new BufferedReader(new InputStreamReader(is));
+                rd.close();
+            } catch (NullPointerException nullPointer) {
+                listener.getLogger().println("Remote Jenkins server returned empty response or invalid JSON - also the Error stream is not available. We just do nothing and can still proceed with the remote build.");
+                return null;
+            }
+            
             BufferedReader rd = new BufferedReader(new InputStreamReader(is));
```

Comment:
I think you can surround just this line with a try-catch block. Only put `BufferedReader rd = null;` before.

Reply:
I am not sure as the lifetime of objects within a try block is local to the try block. This means other commands that are not part of the try block "do not see the object".

Reply:
Variable is not local to a scope if it was declared outside the scope. Therefore it should work:
```
BufferedReader rd = null;
try {
    rd = ....
} ...
```